cypress-bdd.json

```diff
@@ -0,0 +1,17 @@
+{
+  "baseUrl": "https://kiali-istio-system.apps.pmdown4.0216-eg3.fw.rhcloud.com",
```

Comment:
Can you change this to something else by default? Something like `http://localhost:3000`? The base url will likely need to be updated for whoever is running the tests through the cypress env var or through a command line flag.

Reply:
+1 to the suggested URL, it works well when already using `yarn start`.

Reply:
changed to suggested value